kuma/core/whitenoise/KumaDjangoWhiteNoise.py

```diff
@@ -0,0 +1,16 @@
+from whitenoise.django import DjangoWhiteNoise
+
+
+class KumaDjangoWhiteNoise(DjangowhiteNoise):
+
+    # Custom MIME type settings
+    EXTRA_MIMETYPES_KUMA = (
+            ('application/font-woff', '.woff'),
+            ('application/font-woff2', '.woff2'),)
+
+    # Use our custom MIME type settings, together with the default settings, in
+    # WhiteNoise. In the case of a tie (our settings conflicting with the
+    # defaults), use ours.
+    EXTRA_MIMETYPES = tuple({e[1]: e for e in reversed(
+            EXTRA_MIMETYPES_KUMA + DjangoWhiteNoise.EXTRA_MIMETYPES
+        )}.values())
```

Comment:
I'm sure there there's a better way to do this. But this works, and it gives a sense of what I'm going for.